back/src/users/typeDefs/private/user.queries.graphql

```diff
@@ -11,4 +11,12 @@ type Query {
   par son hash
   """
   invitation(hash: String!): Invitation
+
+
+  """
+  USAGE INTERNE
+  Vérifie la validité d'un lien de changement de mot de passe
+  par son hash
+  """
+  resetPassword(hash: String!): Boolean!
```

Comment:
C'est un peu étrange une query qui fait une vérification et retourne un booléen. J'aurai plutôt imaginé une query qui retourne la demande de réinitialisation du mot de passe ou "not found" quand elle n'existe pas ou qu'elle a expiré.